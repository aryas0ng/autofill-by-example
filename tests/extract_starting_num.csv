6649 N Blue Gum St,6649
639 Main St,639
86 Nw 66th St,86
74 S Westgate St,74
34 Center St,
3 Mcauley Dr,
7 Eads St,
7 W Jackson Blvd,
5 Boston Ave ,
228 Runamuck Pl ,
2371 Jerrold Ave,
3273 State St,
25 E 75th St,
98 Connecticut Ave Nw,
56 E Morehead St,
366 South Dr,
69734 E Carrillo St,
322 New Horizon Blvd,
45 E Liberty St,
394 Manchester Blvd,
6 S 33rd St,
6 Greenleaf Ave,
618 W Yakima Ave,