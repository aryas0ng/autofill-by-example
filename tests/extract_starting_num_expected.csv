6649 N Blue Gum St,6649
639 Main St,639
86 Nw 66th St,86
74 S Westgate St,74
34 Center St,34
3 Mcauley Dr,3
7 Eads St,7
7 W Jackson Blvd,7
5 Boston Ave ,5
228 Runamuck Pl ,228
2371 Jerrold Ave,2371
3273 State St,3273
25 E 75th St,25
98 Connecticut Ave Nw,98
56 E Morehead St,56
366 South Dr,366
69734 E Carrillo St,69734
322 New Horizon Blvd,322
45 E Liberty St,45
394 Manchester Blvd,394
6 S 33rd St,6
6 Greenleaf Ave,6
618 W Yakima Ave,618